content/docs/2019-06-23-introducing-mlrplayground.Rmd

```diff
@@ -0,0 +1,120 @@
+---
+title: "Introducing mlrPlayground"
+authors: ["sebastian-gruber"]
+date: "2019-06-23"
+output:
+  blogdown::html_page:
+    toc: true
+categories:
+  - r-bloggers
+tags:
+  - shiny
+  - visualization
+  - mlr
+  - teaching
+  - classification
+  - regression
+  - hyperparameter
+---
+
+<img src="https://i.imgur.com/MPEwCJ2.gif" width=800 />
+<br/>
+
+## First of all
+You may ask yourself how is this name ‘mlrPlayground’ even justified?
+What a madman puts two such opposite terms in a single word and expects people to take him seriously?
+I assume most of you know ‘mlr’, for those who don’t: it’s a framework for machine learning tasks offering a huge variety of different learners based on the best packages available in R, it’s about productivity, simplicity, and extendability.
```

Comment:
its's -> It is

"a huge variety of different learners based on the best packages available in R" -> Let's just say it offers a lot of tools to simplify machine-learning tasks in R

"simplicity, and extendability." maybe we should leave the first word out for mlr ;). The second should be phrased differently.